src/CV8ScriptRuntime.h

```diff
@@ -127,4 +127,36 @@ class CV8ScriptRuntime : public alt::IScriptRuntime
 			return result;
 		}
 	};
+
+	using ResourcesLoadedCallback = void (*)(v8::Local<v8::ScriptOrModule> referrer, v8::Local<v8::String> specifier, const void* promise);
+
+	class ResourcesLoadedResult
```

Comment:
Could be named DynamicImport cause this class (at least for now) is quite specific to dynamic imports